src/commands/du.rs

```diff
@@ -285,13 +298,17 @@ fn glob_err_into(e: GlobError) -> ShellError {
 impl From<DirInfo> for Value {
     fn from(d: DirInfo) -> Self {
         let mut r: IndexMap<String, Value> = IndexMap::new();
+        r.insert(
+            "name".to_string(),
+            UntaggedValue::string(d.name).into_untagged_value(),
+        );
         r.insert(
             "size".to_string(),
             UntaggedValue::bytes(d.size).into_untagged_value(),
         );
         r.insert(
-            "name".to_string(),
-            UntaggedValue::string(d.name).into_untagged_value(),
+            "blocks".to_string(),
+            UntaggedValue::int(d.blocks).into_untagged_value(),
```

Comment:
Compare/contrast:

```
 12 │ target │   4.9 GB │ 2414776832 │ [table 2 rows]
```

vs using `::bytes`:

```
 12 │ target │   5.3 GB │   2.6 GB │ [table 2 rows]
```

Bit closer to what I was after :)